src/main/java/org/broadinstitute/dsm/model/ups/UPSActivity.java

```diff
@@ -8,4 +12,43 @@
     UPSStatus status;
     String date;
     String time;
+
+    public UPSActivity() {}
```

Comment:
Adding explicit no-arg constructors here since this looks like it's used in various reflection based serialization/deserialization tools.